server/core/src/client_api_allowlist.cpp

```diff
@@ -123,7 +123,7 @@ namespace irods::client_api_allowlist
             RM_COLL_AN,
             MOD_COLL_AN,
             COLL_CREATE_AN,
-            DATA_OBJ_UNLOCK_AN,
+            //DATA_OBJ_UNLOCK_AN,
```

Comment:
i think this can just be removed outright... no need to have a placeholder here

Reply:
Agreed.